### DISCUSSION TITLE
Error running inference: file does not exist

### DISCUSSION THREAD

**Discussion by esther-van-pelt** [MIC-DKFZ/nnUNet#1023] [Q&A] [answered]
Hi, 

I am trying to run inference on a trained model, but I am getting error messages that none of my files exist.

This is the command I use for prediction:
nnUNet_predict -o /mnt/beta/egvpelt/inference_results/502/tryout/ -i mnt/beta/egvpelt/nnUNet_raw_data_base/nnUNet_raw_data/Task502_AAA/imagesTs/ -t 502 -m 3d_lowres

This is the last part of the error messages I get (the first part about the file missing in the input folder is repeated for every single image):
![image](https://user-images.githubusercontent.com/71495755/163419638-ee6f8a3b-b51d-4bd9-855d-41ec63980d65.png)
The final error is about the file missing in the output folder, but I assume this is due to the fact that it wasn't found in the input folder in the first place, and thus the preprocessing step is skipped.

However, all images are in the input folder:
![image](https://user-images.githubusercontent.com/71495755/163425454-1c184129-0f47-4d82-aece-d45b1697482e.png)

I hope you can help!

**Comment by XuYinFeng1998** [comment, marked as answer]
I have encountered similar problems. The reason for this problem is that my file permissions are wrong (read-only). You can try to open the read-write permissions of the file.

**Reply by esther-van-pelt** [reply]
Thanks a lot, that helps indeed!

**Comment by FabianIsensee** [comment]
you were missing a `/` in your command:  `-i mnt/beta/egvpelt/nnUNet_raw_data_base/nnUNet_raw_data/Task502_AAA/imagesTs/`. There is none before mnt!
